client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -10,6 +10,8 @@ import { BASE_STALE_TIME } from 'calypso/state/initial-state';
 
 const reviewsApiBase = '/sites/marketplace.wordpress.com/comments';
 const reviewsApiNamespace = 'wp/v2';
+const reviewsStatsApiBase = '/sites/marketplace.wordpress.com/marketplace/reviews';
+const reviewsStatsNamespace = 'wpcom/v2';
```

Comment:
I think the baseUrl for reviews is going to be same. Is it possible to abstract the `reviewsApiBase` here? Something like as suggested. I understand that would require some changes to other endpoint urls as well. 

```suggestion
const reviewsApiBase = '/sites/marketplace.wordpress.com';
const reviewsApiNamespace = 'wp/v2';
const reviewsStatsApiBase =  `${reviewsApiBase}/marketplace/reviews`;
const reviewsStatsNamespace = 'wpcom/v2';
```

Reply:
I agree that we could reuse parts of the URL that the share of the request; however, we need to keep `reviewsApiBase` intact or update it in all places it's used. Something like this would work:

```suggestion
const apiBase = '/sites/marketplace.wordpress.com';
const reviewsApiBase = `${apiBase}/comments`;
const reviewsApiNamespace = 'wp/v2';
const reviewsStatsApiBase = `${apiBase}/marketplace/reviews`;
const reviewsStatsNamespace = 'wpcom/v2';
```
This is all non-blocking IMO too.